### PR TITLE
fix(ios): video player natural size property and event

### PR DESCRIPTION
**Pull request by drauggres**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-27234

Since 7.0.0 VideoPlayers property `naturalSize` and event `naturalSizeAvailable` are broken on iOS. They return video layer size instead of "movie" size.

P.S. I'm not obj-c developer, so please check carefully (it's working for my case).

### CODE REVIEW

File: iphone/Classes/TiMediaVideoPlayerProxy.m [hunk 1]
```diff
@@ -702,9 +702,13 @@ - (NSNumber *)repeatMode
 
 - (id)naturalSize
 {
+  CGSize size = CGSizeZero;
+  if (movie && [movie player] && [[movie player] currentItem]) {
```
Comment (hansemannn) [inline iphone/Classes/TiMediaVideoPlayerProxy.m]: ObjC supports optional chaining, so `if (movie.player.currentItem)` should work fine, even if `movie.player` is `nil.`

Comment (drauggres) [reply iphone/Classes/TiMediaVideoPlayerProxy.m]: I was looking for Axway coding style conventions about "dots vs brackets" (didn't find any), but instead I found [this](https://docs.axway.com/bundle/Titanium_SDK_allOS_en/page/objective-c_and_objective-c___coding_standards.html):
>
> It is illegal to use a statement such as {{ if (objcObject) { ... } }}. Instead directly compare to nil, only where required.

So, is `if (movie.player.currentItem)` OK?

Comment (hansemannn) [reply iphone/Classes/TiMediaVideoPlayerProxy.m]: Well, it's really up to the developer. Nowadays, the dot-notation is preferred (introduced in ObjC 2.0+), but it's not part of the linting / clang-format. In regards to the nil check, the `movie.player.currentItem` is more strict, like `===` on JS.

Comment (drauggres) [reply iphone/Classes/TiMediaVideoPlayerProxy.m]: OK. Thank you, `movie.player.currentItem` it is.

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Comment by vijaysingh-axway** [comment]
Thanks @drauggres for PR and @hansemannn  for review.

**Comment by keerthi1032** [comment]
@vijaysingh-axway with the fix  naturalSize & naturalSizeAvailable  does not return values .shows blank .added screenshot for reference. can you please look at it 
Screen Shot 2019-08-05 at 2 40 42 PM

**Comment by drauggres** [comment]
@keerthi1032 Actually it is a problem with `console.log`.
Please try with
```javascript
    console.log(e.type + ' e.naturalSize ' + JSON.stringify(e.naturalSize));
    console.log('player.naturalSize ' + JSON.stringify(e.source.naturalSize));
```

**Comment by drauggres** [comment]
Also please consider opening a new ticket for `console.log` bug.
>`console.log( message )`
>Log a message at the `info` level.
>The message to log can either be a single argument, or **any number of arguments**, which will be converted to strings and then concatenated together with a space character.

**Comment by keerthi1032** [comment]
Thanks .it is working now.

**Pull request merged**